css/styles.css

```diff
@@ -88,4 +88,80 @@ header {
   padding: 8px;
   margin-right: 32px;
   color: #777;
-}
\ No newline at end of file
+}
+
+.spicestore {
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-start;
+}
+
+.spicestore h1 {
+  font-size: 68px;
+  color: #4c884c;  
+  margin-top: -1px;
+  margin-bottom: -1px;
+  cursor: pointer;
+  padding-top: 5px;
+  flex-shrink: 0;
+}
+
+.word2 {
+color: #ddd;
+cursor: pointer;
```

Comment:
this should not be needed as the parent already got cursor: pointer;. do double check.